sql/2019/04_Media/04_19.sql

```diff
@@ -0,0 +1,71 @@
+/*
+04_19 
+
+10TB
+
+only tested on the sample response body samples
+
+This pulls out every parameter from the video tag and totals them.
```

Comment:
In hindsight, this would have been a lot more efficient as a custom metric. 😬

Reply:
:)